packages/api/src/models/monitoring/operation.ts

```diff
@@ -37,23 +37,23 @@ export class Operation extends Model<Operation> {
   href: string;
   labelKey: string;
 
-  constructor(operation: Operation) {
+  constructor(operation: Partial<Operation>) {
     super();
+    this.value = operation.value || '';
+    this.status = operation.status || OperationStatus.INFORMATION;
```

Comment:
I am not sure about this and the next default. As a suggestion, we could throw an error if they are not provided.

Reply:
Indeed. These defaults should not be here, but in the mapper if needed at all. The only reason those are here I think is that the operation argument is passed as a Partial which makes its properties optional. I'll fix this.
Regarding the suggestion to throw error if these are missing, you can see the `toEnum` utility used in the mapper and notice that it actually is throwing error in such cases.
Is that good enough?